wallet/src/actors/app/params.rs

```diff
@@ -9,8 +9,14 @@ use crate::actors;
 pub struct Params {
     pub testnet: bool,
     pub worker: Addr<actors::Worker>,
-    pub client: Addr<JsonRpcClient>,
+    pub client: NodeClient,
     pub server_addr: SocketAddr,
     pub session_expires_in: Duration,
     pub requests_timeout: Duration,
 }
+
+#[derive(Clone)]
+pub struct NodeClient {
+    pub url: String,
+    pub actor: Addr<JsonRpcClient>,
```

Comment:
`Addr` is just an `Arc` internally so it be cloned easily, but `String` will perform a deep clone. So `url` should probably be `Arc<str>`. Also I think `url` is never used, so it can be removed.

Reply:
We could also simply use `Arc<NodeClient>` when clones are needed instead.

Reply:
True, it's never used.

Reply:
It's used inside a `json!` macro in `notify_client`.